src/data/db2_secondHalfAppSettings.js

```diff
@@ -0,0 +1,69 @@
+import { APP_SETTING_NAMES, QUESTION_TYPES } from '../config/constants';
+
+export const db2_secondHalfQuestionsList = ['id6', 'id7', 'id8', 'id9', 'id10'];
+export const db2_secondHalfAppSettings = {
```

Comment:
Improve naming and file name?

Reply:
I removed this file, and put all of its content directly into the `db` file.